### DISCUSSION TITLE
Question set is not appearing in the review list after sending it for review.

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-inQuiry/Community#58] [Bugs]
The question set is not appearing in the review list after sending it for review

**Comment by pallakartheekreddy** [comment]
@sachinb80 Where are you refereeing? Can you please share more details or Screenshot?

**Reply by sachinb80** [reply]
Yes sure
![review issue 1](https://user-images.githubusercontent.com/109504832/216566681-1bae69a4-1251-4dc3-bf43-fb84d498ab6a.png)
![review issue 2](https://user-images.githubusercontent.com/109504832/216566707-84032458-2f9e-4394-af59-071fb5d2be00.png)
Steps to reproduce
Login with content creator id
Create question set
Send it for review
login with content reviewer id
go to workspace and click on up for review
check screenshots for more info

**Reply by pallakartheekreddy** [reply]
@sachinb80 This issue related to sunbird ed building block. I guess review workflow for Question Sets not enabled in sunbird ed. Please create this bug in below discussion thread. 
https://github.com/Sunbird-Ed/Community/discussions
@vinukumar-vs @Krishnaj20 Please take a look at this thread.

**Reply by sachinb80** [reply]
ok
